### DISCUSSION TITLE
Sending TSlim Pump Data to XDrip

### DISCUSSION THREAD

**Discussion by acnofsinger** [NightscoutFoundation/xDrip#2768] [Q&A]
I use TASKER with AUTOTOOLS plugin to open T:Connect app on my android and use the UI Query function to get all of my pump data (Battery, Basal Rate, Cartridge level, IOB).  I send these variable to my Wear OS watch every 5 minutes.  I would like to know if I can use Tasker to send these variable to XDrip.  This would be helpful to trend the actual basal rate being used by my CONTROLIQ on my pump.  Thanks!!

**Comment by Navid200** [comment]
Would you please have a look at the following and let me know if it is what you are looking for?
https://github.com/NightscoutFoundation/xDrip/issues/904
